<robot name="box">
 	<link name="base_link">
		<visual>
		    <origin xyz="0 0 0" rpy="0 0 0"/>
          <axis xyz="0 0 0" />
			<geometry>
				<box size="0.08 0.1 0.18"/>
			</geometry>
			<material name="Cyan">
	       		<color rgba="0 1.0 1.0 1.0"/>
	     	</material>
		</visual>	
	</link>
    <link name="tag36_11">
    <visual>
      <origin xyz="0 0 0.00000"/>
      <geometry>
        <mesh filename="package://gerard_bauzil/meshes/apriltag_36h11/tag36_11_00100.dae" scale="0.0573 0.0573 1."/>
      </geometry>
    </visual>
    </link>
   
   <joint name="to_tag" type="fixed">
    <parent link="base_link"/>
    <child link="tag36_11"/>
    <origin xyz="0.05 0 0"/>
   </joint>
  
</robot>